bears/general/ImageDimensionBear.py

```diff
@@ -0,0 +1,70 @@
+import os
+import yaml
+import shutil
+
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from dependency_management.requirements.GemRequirement import GemRequirement
+from dependency_management.requirements.PipRequirement import PipRequirement
+from sarge import run, Capture
+
+
+class ImageDimensionBear(GlobalBear):
+    """
+    Checks the dimension of an image us a gem.
+
+    More information is available at <github.com/Abhi2424shek/img_checker>
+    """
+
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    REQUIREMENTS = {GemRequirement(' img_checker'),
+                    PipRequirement('pyyaml', '3.12')}
+    LICENSE = 'AGPL-3.0'
+    CAN_FIX = {'Image Dimension'}
+
+    @classmethod
+    def check_prerequisites(cls):
+        if shutil.which('img_checker') is None:
+            return 'img_checker is not installed.'
+        return True
+
+    def run(self,
+            image_file,
```

Comment:
you need to define the types of these parameters using Python 3 syntax.
See https://github.com/coala/coala-bears/blob/master/bears/coffee_script/CoffeeLintBear.py#L63 for an example